specs/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "test_helper"
+
+describe "User class" do
+  describe "self.list" do
+    it "can return all users" do
+      VCR.use_cassette("slack_user") do
```

Comment:
You should include tests for `User`'s entire public interface: `User.list`, the constructor, `User#details`, and `User#send_message`.